Make install fails on java

### Version
Current git head

### Describe the bug

git clone https://github.com/zeroc-ice/ice
cd ice
export JAVA_HOME=/opt/jdk-21.0.10
make -j 16
make install

comet:/usr/src/ice# make install
making all in cpp
make[1]: Entering directory '/usr/src/ice/cpp'
Installing CMake files for x86_64
make[1]: Leaving directory '/usr/src/ice/cpp'
making all in java
make[1]: Entering directory '/usr/src/ice/java'
make[1]: *** No rule to make target 'install'.  Stop.
make[1]: Leaving directory '/usr/src/ice/java'
make: *** [Makefile:21: install] Error 1


### Expected behavior

I would have expected "make install" to work if the initial make returned error code 0.

### Additional context

As far as I can tell the build was completely successful:

comet:/usr/src/ice# make -j 16
making all in cpp
make[1]: Entering directory '/usr/src/ice/cpp'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/usr/src/ice/cpp'
making all in java
make[1]: Entering directory '/usr/src/ice/java'
./gradlew -x checkStyleMain -x checkStyleTest build
Reusing configuration cache.

BUILD SUCCESSFUL in 453ms
84 actionable tasks: 30 executed, 54 up-to-date
Configuration cache entry reused.
make[1]: Leaving directory '/usr/src/ice/java'
making all in python
make[1]: Entering directory '/usr/src/ice/python'
make[2]: Entering directory '/usr/src/ice/python/python'
make[2]: Nothing to be done for 'generate-srcs'.
make[2]: Leaving directory '/usr/src/ice/python/python'
make[2]: Entering directory '/usr/src/ice/python/test'
make[2]: Nothing to be done for 'generate-srcs'.
make[2]: Leaving directory '/usr/src/ice/python/test'
make[2]: Entering directory '/usr/src/ice/python/python'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/usr/src/ice/python/python'
make[2]: Entering directory '/usr/src/ice/python/test'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/usr/src/ice/python/test'
make[1]: Leaving directory '/usr/src/ice/python'
making all in ruby
make[1]: Entering directory '/usr/src/ice/ruby'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/usr/src/ice/ruby'
making all in php
make[1]: Entering directory '/usr/src/ice/php'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/usr/src/ice/php'
making all in js
make[1]: Entering directory '/usr/src/ice/js'
npm install

up to date, audited 550 packages in 986ms

108 packages are looking for funding
  run `npm fund` for details

4 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
npm run build

> build
> gulp

[12:13:22] Using gulpfile /usr/src/ice/js/gulpfile.js
[12:13:22] Starting 'default'...
[12:13:22] Starting 'build'...
[12:13:22] Starting 'dist'...
[12:13:22] Starting 'Ice:generate'...
[12:13:22] Starting 'Glacier2:generate'...
[12:13:22] Starting 'IceBox:generate'...
[12:13:22] Starting 'IceStorm:generate'...
[12:13:22] Starting 'IceGrid:generate'...
[12:13:22] Finished 'IceBox:generate' after 11 ms
[12:13:22] Finished 'IceStorm:generate' after 15 ms
[12:13:22] Finished 'Glacier2:generate' after 19 ms
[12:13:22] Finished 'Ice:generate' after 27 ms
[12:13:22] Finished 'IceGrid:generate' after 42 ms
[12:13:22] Finished 'dist' after 43 ms
[12:13:22] Starting 'test'...
[12:13:22] Starting 'ice:bundle'...
[12:13:23] Finished 'ice:bundle' after 526 ms
[12:13:23] Starting 'test:common:build'...
[12:13:23] Finished 'test:common:build' after 20 ms
[12:13:23] Starting 'Ice_moduleAggregation_relative:build'...
[12:13:23] Finished 'Ice_moduleAggregation_relative:build' after 6.58 ms
[12:13:23] Starting 'Ice_moduleAggregation_module:build'...
[12:13:23] Finished 'Ice_moduleAggregation_module:build' after 7.05 ms
[12:13:23] Starting 'Ice_moduleAggregation_external:build'...
[12:13:23] Finished 'Ice_moduleAggregation_external:build' after 3.67 ms
[12:13:23] Starting 'Ice_adapterDeactivation:build'...
[12:13:24] Finished 'Ice_adapterDeactivation:build' after 1.36 s
[12:13:24] Starting 'Ice_ami:build'...
[12:13:25] Finished 'Ice_ami:build' after 1.11 s
[12:13:25] Starting 'Ice_binding:build'...
[12:13:26] Finished 'Ice_binding:build' after 976 ms
[12:13:26] Starting 'Ice_defaultValue:build'...
[12:13:27] Finished 'Ice_defaultValue:build' after 898 ms
[12:13:27] Starting 'Ice_defaultServant:build'...
[12:13:28] Finished 'Ice_defaultServant:build' after 979 ms
[12:13:28] Starting 'Ice_enums:build'...
[12:13:29] Finished 'Ice_enums:build' after 878 ms
[12:13:29] Starting 'Ice_exceptions:build'...
[12:13:30] Finished 'Ice_exceptions:build' after 892 ms
[12:13:30] Starting 'Ice_facets:build'...
[12:13:31] Finished 'Ice_facets:build' after 860 ms
[12:13:31] Starting 'Ice_idleTimeout:build'...
[12:13:32] Finished 'Ice_idleTimeout:build' after 889 ms
[12:13:32] Starting 'Ice_inactivityTimeout:build'...
[12:13:33] Finished 'Ice_inactivityTimeout:build' after 870 ms
[12:13:33] Starting 'Ice_info:build'...
[12:13:33] Finished 'Ice_info:build' after 851 ms
[12:13:33] Starting 'Ice_inheritance:build'...
[12:13:34] Finished 'Ice_inheritance:build' after 873 ms
[12:13:34] Starting 'Ice_location:build'...
[12:13:35] Finished 'Ice_location:build' after 972 ms
[12:13:35] Starting 'Ice_middleware:build'...
[12:13:36] Finished 'Ice_middleware:build' after 866 ms
[12:13:36] Starting 'Ice_moduleAggregation:build'...
[12:13:37] Finished 'Ice_moduleAggregation:build' after 860 ms
[12:13:37] Starting 'Ice_objects:build'...
[12:13:38] Finished 'Ice_objects:build' after 947 ms
[12:13:38] Starting 'Ice_operations:build'...
[12:13:39] Finished 'Ice_operations:build' after 1.21 s
[12:13:39] Starting 'Ice_optional:build'...
[12:13:40] Finished 'Ice_optional:build' after 961 ms
[12:13:40] Starting 'Ice_properties:build'...
[12:13:41] Finished 'Ice_properties:build' after 848 ms
[12:13:41] Starting 'Ice_proxy:build'...
[12:13:42] Finished 'Ice_proxy:build' after 944 ms
[12:13:42] Starting 'Ice_retry:build'...
[12:13:43] Finished 'Ice_retry:build' after 934 ms
[12:13:43] Starting 'Ice_servantLocator:build'...
[12:13:44] Finished 'Ice_servantLocator:build' after 872 ms
[12:13:44] Starting 'Ice_slicing_exceptions:build'...
[12:13:45] Finished 'Ice_slicing_exceptions:build' after 828 ms
[12:13:45] Starting 'Ice_slicing_objects:build'...
[12:13:45] Finished 'Ice_slicing_objects:build' after 946 ms
[12:13:45] Starting 'Ice_stream:build'...
[12:13:46] Finished 'Ice_stream:build' after 851 ms
[12:13:46] Starting 'Ice_timeout:build'...
[12:13:47] Finished 'Ice_timeout:build' after 809 ms
[12:13:47] Starting 'Ice_scope:build'...
[12:13:48] Finished 'Ice_scope:build' after 852 ms
[12:13:48] Starting 'Glacier2_router:build'...
[12:13:49] Finished 'Glacier2_router:build' after 897 ms
[12:13:49] Starting 'Slice_escape:build'...
[12:13:50] Finished 'Slice_escape:build' after 819 ms
[12:13:50] Starting 'Slice_macros:build'...
[12:13:51] Finished 'Slice_macros:build' after 801 ms
[12:13:51] Starting 'Slice_moduleMapping:build'...
[12:13:51] Finished 'Slice_moduleMapping:build' after 819 ms
[12:13:51] Starting 'Slice_moduleName:build'...
[12:13:52] Finished 'Slice_moduleName:build' after 1.01 s
[12:13:52] Starting 'Ice_adapterDeactivation:copy:assets'...
[12:13:52] Finished 'Ice_adapterDeactivation:copy:assets' after 2.26 ms
[12:13:52] Starting 'Ice_ami:copy:assets'...
[12:13:52] Finished 'Ice_ami:copy:assets' after 1.45 ms
[12:13:52] Starting 'Ice_binding:copy:assets'...
[12:13:52] Finished 'Ice_binding:copy:assets' after 1.33 ms
[12:13:52] Starting 'Ice_defaultValue:copy:assets'...
[12:13:52] Finished 'Ice_defaultValue:copy:assets' after 1.4 ms
[12:13:52] Starting 'Ice_defaultServant:copy:assets'...
[12:13:52] Finished 'Ice_defaultServant:copy:assets' after 2.06 ms
[12:13:52] Starting 'Ice_enums:copy:assets'...
[12:13:52] Finished 'Ice_enums:copy:assets' after 1.26 ms
[12:13:52] Starting 'Ice_exceptions:copy:assets'...
[12:13:52] Finished 'Ice_exceptions:copy:assets' after 1.22 ms
[12:13:52] Starting 'Ice_facets:copy:assets'...
[12:13:52] Finished 'Ice_facets:copy:assets' after 1.23 ms
[12:13:52] Starting 'Ice_idleTimeout:copy:assets'...
[12:13:52] Finished 'Ice_idleTimeout:copy:assets' after 1.1 ms
[12:13:52] Starting 'Ice_inactivityTimeout:copy:assets'...
[12:13:52] Finished 'Ice_inactivityTimeout:copy:assets' after 1.07 ms
[12:13:52] Starting 'Ice_info:copy:assets'...
[12:13:52] Finished 'Ice_info:copy:assets' after 1.09 ms
[12:13:52] Starting 'Ice_inheritance:copy:assets'...
[12:13:52] Finished 'Ice_inheritance:copy:assets' after 981 μs
[12:13:52] Starting 'Ice_location:copy:assets'...
[12:13:52] Finished 'Ice_location:copy:assets' after 970 μs
[12:13:52] Starting 'Ice_middleware:copy:assets'...
[12:13:52] Finished 'Ice_middleware:copy:assets' after 1.03 ms
[12:13:52] Starting 'Ice_moduleAggregation:copy:assets'...
[12:13:52] Finished 'Ice_moduleAggregation:copy:assets' after 967 μs
[12:13:52] Starting 'Ice_objects:copy:assets'...
[12:13:52] Finished 'Ice_objects:copy:assets' after 1 ms
[12:13:52] Starting 'Ice_operations:copy:assets'...
[12:13:52] Finished 'Ice_operations:copy:assets' after 998 μs
[12:13:52] Starting 'Ice_optional:copy:assets'...
[12:13:52] Finished 'Ice_optional:copy:assets' after 1.02 ms
[12:13:52] Starting 'Ice_properties:copy:assets'...
[12:13:52] Finished 'Ice_properties:copy:assets' after 1.09 ms
[12:13:52] Starting 'Ice_proxy:copy:assets'...
[12:13:52] Finished 'Ice_proxy:copy:assets' after 982 μs
[12:13:52] Starting 'Ice_retry:copy:assets'...
[12:13:52] Finished 'Ice_retry:copy:assets' after 951 μs
[12:13:52] Starting 'Ice_servantLocator:copy:assets'...
[12:13:52] Finished 'Ice_servantLocator:copy:assets' after 964 μs
[12:13:52] Starting 'Ice_slicing_exceptions:copy:assets'...
[12:13:52] Finished 'Ice_slicing_exceptions:copy:assets' after 989 μs
[12:13:52] Starting 'Ice_slicing_objects:copy:assets'...
[12:13:52] Finished 'Ice_slicing_objects:copy:assets' after 971 μs
[12:13:52] Starting 'Ice_stream:copy:assets'...
[12:13:52] Finished 'Ice_stream:copy:assets' after 957 μs
[12:13:52] Starting 'Ice_timeout:copy:assets'...
[12:13:52] Finished 'Ice_timeout:copy:assets' after 945 μs
[12:13:52] Starting 'Ice_scope:copy:assets'...
[12:13:52] Finished 'Ice_scope:copy:assets' after 939 μs
[12:13:52] Starting 'Glacier2_router:copy:assets'...
[12:13:52] Finished 'Glacier2_router:copy:assets' after 940 μs
[12:13:52] Starting 'Slice_escape:copy:assets'...
[12:13:52] Finished 'Slice_escape:copy:assets' after 950 μs
[12:13:52] Starting 'Slice_macros:copy:assets'...
[12:13:52] Finished 'Slice_macros:copy:assets' after 919 μs
[12:13:52] Starting 'Slice_moduleMapping:copy:assets'...
[12:13:52] Finished 'Slice_moduleMapping:copy:assets' after 918 μs
[12:13:52] Starting 'Slice_moduleName:copy:assets'...
[12:13:52] Finished 'Slice_moduleName:copy:assets' after 989 μs
[12:13:52] Finished 'test' after 30 s
[12:13:52] Starting 'test:unplugin'...
[12:13:55] Finished 'test:unplugin' after 2.13 s
[12:13:55] Finished 'build' after 32 s
[12:13:55] Finished 'default' after 32 s
make[1]: Leaving directory '/usr/src/ice/js'

--- a/java/Makefile
+++ b/java/Makefile
@@ -15,5 +15,8 @@ tests:
 srcs:
 	$(GRADLE) $(GRADLEARGS) dist
 
+install:
+# Nothing to install for Java.
+
 distclean clean:
 	$(GRADLE) $(GRADLEARGS) clean

